crates/core/src/models/block.rs

```diff
@@ -143,7 +143,7 @@ impl ConvertibleStarknetBlock for BlockWithTxHashes {
 
         // Bloom is a byte array of length 256
         let logs_bloom = Bloom::default();
-        let extra_data = Bytes::from(&[0]);
+        let extra_data = Bytes::default();
```

Comment:
yeah, this is great. using default in as many places as we can brings warmth to my soul